coreplugins/dronedb/api_views.py

```diff
@@ -273,7 +273,17 @@ def ddb_cleanup(sender, task_id, **kwargs):
     # When a task is re-processed, make sure we can re-share it if we shared a task previously
 
     logger.info("Cleaning up DroneDB datastore for task {}".format(str(task_id)))
-    #ds.del_key(get_key_for(task_id, "status"))
+
+    task = models.Task.objects.get(id=task_id)
```

Comment:
This is not working. I need to get the project id from the task id. I'm using the ORM but it seems that no field `id` is found in the `project` object. What am I doing wrong? @pierotofy